DigitalLearningSolutions.Web/ViewModels/LearningPortal/RecommendedLearning/SearchableRecommendedResourceViewModel.cs

```diff
@@ -18,7 +18,8 @@ public SearchableRecommendedResourceViewModel(RecommendedResource recommendedRes
             IsCompleted = recommendedResource.IsCompleted;
             LearningLogItemId = recommendedResource.LearningLogId;
 
-            Rating = GetRecommendationRatingString(recommendedResource.RecommendationScore);
+            Rating = recommendedResource.RecommendationScore >= 100 ? "Essential" :
+                recommendedResource.RecommendationScore >= 40 ? "Recommended" : "Optional";

```

Comment:
Ah, I was actually thinking about making this the get property of Rating and removing its set property, that way if a view model is ever modified for some reason Rating will always be accurate.